### PR TITLE
Jedi cmk ai

### PR DESCRIPTION
**Pull request by VeryEarly**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/applicationinsights/resource-manager/readme.md [hunk 1]
```diff
@@ -324,6 +324,28 @@ input-file:
 - Microsoft.Insights/preview/2020-03-01-preview/componentLinkedStorageAccounts_API.json
 ```
 
+### Tag: package-2020-04
```
Comment (MyronFanQiu) [inline specification/applicationinsights/resource-manager/readme.md:327]: what's this tag used for? If it's only for PS, we may name it as a specific name.

Comment (VeryEarly) [reply specification/applicationinsights/resource-manager/readme.md:327]: yes, it's for PS to include APIs 2020-03-01-preview and 2020-02-02-preview.

what kind of name can I use? can you give some examples?

Comment (MyronFanQiu) [reply specification/applicationinsights/resource-manager/readme.md:327]: like `package-2020-04-powershell`? or add the information into the description.

**Review by mmyyrroonn — APPROVED** [review]

**Review by olterner — APPROVED** [review]

File: specification/applicationinsights/resource-manager/readme.md (hunk 1 above)
Comment (VeryEarly) [reply specification/applicationinsights/resource-manager/readme.md:327]: description added

**Pull request merged**